CONTRIBUTING.md

```
@@ -15,16 +15,21 @@ We welcome contributions from everyone. By participating in this project, you ag
-- [Pull Request Guidelines](#pull-request-guidelines)
-- [Code Style Guidelines](#code-style-guidelines)
-- [Debugging Pre-commit Hooks](#debugging-pre-commit-hooks)
-- [Community Guidelines](#community-guidelines)
+## 📋 Table of Contents
+
+- [How Can I Contribute?](#-how-can-i-contribute)
+- [Content vs. Code](#-content-vs-code)
+- [Coding with AI](#-coding-with-ai)
+- [Getting Started](#-getting-started)
+- [Development Workflow](#-development-workflow)
+- [The One Rule That Will Bite You](#-the-one-rule-that-will-bite-you)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)